themes/coderdojoschoeneweide/layouts/_default/baseof.html

```diff
@@ -3,6 +3,31 @@
 <html lang="{{.Site.LanguageCode}}" dir="{{.Site.Language.LanguageDirection}}">
 {{ block "head.html" . }}
 <head>
+
+    <div class="abod-lang-2">
+        {{ range .Translations }}
+          {{ if eq .Lang "en" }}
+              {{/* <link rel="alternate" hreflang="{{ .Lang }}" href="{{ .Permalink }}" /> */}}
+              <li class="hover:bg-slate-300 p-2">
+                <a href="{{ .Permalink }}">
+                    <img src="https://cdn-icons-png.flaticon.com/32/8281/8281549.png" alt="  ">
```

Comment:
Wir können leider nicht einfach Bilder aus einer fremden Quelle einbinden, da wir das sonst in unserer Datenschutzerklärung berücksichtigen müssten. Könntest Du
 die Icons stattdessen runterladen und direkt als Dateien hinzufügen?
Außerdem müssen wir aufgrund der Lizenz, unter der die Icons stehen, auf die Quelle verweisen. Könntest Du entsprechende Verweise im Impressum ergänzen? Dort steht auch schon ein Verweis für unsere Font-Awesome-Icons :)